File: Assignment_2/Calculator.js
var input = process.argv.slice(2).join(' ');

var expr = input.split(' ', 5);
var int1 = parseInt(expr[0], 10);
var operand = expr[1];
var int2 = parseInt(expr[2], 10);
var out;

if(operand == "+") {
  out = int1 + int2;
}else if(operand == "-") {
  out = int1 - int2;
}else if(operand == "*") {
  out = int1 * int2;
}else {
  out = int1 / int2;
}

console.log(input + " = " + out.toString(10));

File: Assignment_2/Calculator.py
import sys

args  = str(sys.argv[1])

args_split = args.split()

int1 = int(args_split[0])
int2 = int(args_split[2])
operand = args_split[1]

if operand == "+":
    out = int1 + int2
elif operand == "-":
    out = int1 - int2
elif operand == "*":
    out = int1 * int2
elif operand == "/":
    out = int1 / int2

print(args + " = " + str(out))
